### ISSUE TITLE
Settings page looped up to error 504

### ISSUE THREAD

**Issue by danielwpsouza** [mautic/mautic#10016]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
The main error is 504 after a few minutes Loop. 

For permission testing
I changed the permissions on the "var" and "config" folders to 777.
I changed the permissions of the files in the "config" folder to 777.

This error also happened in Version 3.3.1, I made the update to version 3.3.3 to check if the error would be corrected.

I took the steps in this tutorial: https://johnlinhart.com/blog/uh-oh-mautic-upgrade-was-not-successful.

I have other installations on the same server with the same configurations described below, but only the one with this error.

| Q   | A
| --- | ---
| Mautic version | 3.3.3
| PHP version | 7.3.28
| Browser | Chrome
| Host | AWS
| Server | Apache
| Manipulator | PHP-FPM
| DB in GB | 4GB

### Steps to reproduce
1. Access https://mymautic.com/s/config/edit
 
### Log errors
No errors

###Log Apache
No errors

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by RCheesley** [comment]
Hi there, looks to be a server config issue with your cache - please post this at https://forum.mautic.org/c/support.

**Comment by danielwpsouza** [comment]
Hello how are you?

I removed the error log information, as it was caused by permission incompatible with PHP-FPM.

I can access the Settings Page if I change the Server Timeout to 10 Minutes.
Keep-Alive Timeout = 10 Minutes
Timeout = 3000

![error-mautic](https://user-images.githubusercontent.com/72410310/118186301-f6078b80-b413-11eb-9f97-b69986896748.png)


This installation has 4GB of Database, the other 1GB of Database, this is the only difference, that I see.

What does the Settings page load? Where does it look for the data, in addition to the local.php file?

Looking at the performance of the server / database, they do not reach 10% of vCPU consumption and the memory is always below 50% of consumption.

All pages load well, except the settings page.

As requested, I opened a discussion at: https://forum.mautic.org/t/settings-page-looped-up-to-error-504/19447
